content/blog/getting-started-with-hpe-greenlake-api-for-virtualization.md

```diff
@@ -0,0 +1,412 @@
+
+```json
+{
+    “associatedResources”: [],
+    “childTasks”: [
+        {
+            “name”: “Create Instance”,
+            “resourceUri”: “/data-services/v1beta1/async-operations/286cc55c-8636-4a8a-8428-0a5694f42785”,
+            “type”: “task”
+        }
+    ],
+    “createdAt”: “2024-03-28T01:53:30.395518007Z”,
+    “customerId”: “xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx”
+    “displayName”: “Create Instance: RonD-deploy-CSP-1”,
+    “endedAt”: “2024-03-28T01:54:04.245722969Z”,
+    “error”: null,
+    “estimatedRunningDurationMinutes”: 0,
+    “generation”: 3,
+    “groups”: [
+        {
+            “id”: “eb988b5e2dcb11ec840712b3b5263ef4”,
+            “name”: “Default Group”
+        }
+    ],
+    “healthStatus”: “OK”,
+    “id”: “5f54e591-xxxx-xxx-xxxx-bebadb977f93”,
+    “logMessages”: [
+        {
+            “message”: “Create Instance: RonD-deploy-CSP-1 task is created”,
+            “timestampAt”: “2024-03-28T01:53:30.395526907Z”
+        },
+        {
+            “message”: “Create Instance: RonD-deploy-CSP-1 task is running”,
+            “timestampAt”: “2024-03-28T01:53:30.395529707Z”
+        },
+        {
+            “message”: “Create Instance: RonD-deploy-CSP-1 task is succeeded”,
+            “timestampAt”: “2024-03-28T01:54:04.245712087Z”
+        }
+    ],
+    “name”: “Create Instance: RonD-deploy-CSP-1”,
+    “parentTask”: null,
+    “progressPercent”: 100,
+    “recommendations”: [],
+    “resourceUri”: “/data-services/v1beta1/async-operations/5f54e591-a231-46ce-a2e2-bebadb977f93”,
+    “rootTask”: {
+        “id”: “5f54e591-a231-46ce-a2e2-bebadb977f93”,
+        “name”: “”,
+        “resourceUri”: “/data-services/v1beta1/async-operations/5f54e591-a231-46ce-a2e2-bebadb977f93”,
+        “type”: “task”
+    },
+    “services”: [
+        “private-cloud-business-edition”
+    ],
+    “startedAt”: “2024-03-28T01:53:30.395520016Z”,
+    “state”: “SUCCEEDED”,
+    “subtreeTaskCount”: 1,
+    “suggestedPollingIntervalSeconds”: 30,
+    “type”: “task”,
+    “updatedAt”: “2024-03-28T01:54:04.324786754Z”,
+    “userId”: “xxxx.yyyy@abc.com"
+}
+```
+
+*T﻿he above snippet of response code depicted that execution of `POST /virtual/v1beta1/machine-instance had completed successfully`.*
+
+To progress further, I needed to find the Id that corresponds to the newly created virtual-machine instance in the AWS account using the Create CSP machine instance API. To find the virtual machine Id of the deployed VM, I used a legacy HPE GreenLake Data Services Cloud Console API `GET {baseUrl}/api/v1/csp-machine-instances` to get the list of virtual-machine-instance that exist in that AWS account. From the response of that API, I obtained the VM instance Id of that Virtual Machine that was created in prior example.  
```

Comment:
At the end of the final sentence, have it read:
"...that was created in **the** prior example."